src/js/search/widgets/infinite_scroll.js

```diff
@@ -20,7 +20,11 @@ function renderTemplate(template, res) {
 }
 
 function scrolledNearBottom(el, offset) {
-  return (el.scrollHeight - el.scrollTop) < offset;
+  if(window.innerHeight < offset){
```

Comment:
Which browsers did you test this in? I received a message that the code is failing in the latest Firefox--is this fix working for you there?